cdap-docs/application-templates/source/etl/templates/sinks/batch/snapshotavro.rst

```diff
@@ -0,0 +1,60 @@
+.. meta::
+:author: Cask Data, Inc.
```

Comment:
In both files, this (":author:") needs to be indented so that it matches the copyright label below it.